README.md

```diff
@@ -3,8 +3,8 @@
 Hello! My name is Olubusayo Bolonduro but I just go by **Busayo**.
 
 I am an incoming Genetics, Genomics, and Systems Biology student. My biggest research interests are:
-*Population Genetics
-*Comparative Genomics
-*Human Genetics and Disease
+* Population Genetics
+* Comparative Genomics
+* Human Genetics and Disease
 
 Looking forward to hopefully meeting you all in the fall! Enjoy a clip from [one of the best things that has happened in sports in the last decade!](https://www.youtube.com/watch?v=N-byJvHAXQA)
```

Comment:
I had been trying to embed a youtube video as well, but it wasn't working because I was switching the bracket types. Thanks for showing me how it's done! :)